src/main/java/core/basesyntax/interfaces/Calculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface Calculable {
+    double area();
```

Comment:
```suggestion
    double getArea();
```
method should start with verb